UMLEditor/UMLEditor/Views/CommandLine/CommandLine.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace UMLEditor.Views.CommandLine;
+
+/// <summary>
+/// CommandLine.cs
+/// </summary>
+public class CommandLine
+{
+    private CommandLineController? _controller;
```

Comment:
The controller doesn't need to be nullable. You should initialize this in a constructor.